### ISSUE TITLE
Themely doesnst start

### ISSUE THREAD

**Issue by memeToasty** [ErmanSayin/Themely#121]
Themely doesnt start it stucks at a brown orangy display... I have o3ds homebrew RPwnG
I have the 3dsx file in the 3ds folder. Pls help

**Comment by Zemogiter** [comment]
I have the same issue, New 3DS XL EUR, OFW 11.5, using RPwnG, payload is updated.

**Comment by memeToasty** [comment]
mine is more yellow or brown. I heard if the screen is red u plaxed your starter files wrong. So try to reinstall them

**Comment by CallMehMerel** [comment]
I also have the same issue on an US o3ds using RPwnG. My best guess for this is that themely doesn't work with the latest firmware version, as most of the homebrew applications I was once able to use do not work when I launch them giving the same crash screen

**Comment by That1Guy20XX** [comment]
I have the same problem. Themely 1.4.0 on N3DS 11.5 using RPwnG. Any ideas?

**Comment by CallMehMerel** [comment]
Like I edited into my prevoius comment, according to MrNbaYoh's twitter, a lot of people have concluded that the issue with this crash is that the new payloads arent all that stable and might not be compatable with the coding a lot of old homebrew programs used so those programs will probably have to be recoded to fit the new code of the payloads

**Comment by ErmanSayin** [comment]
most likely an issue by the exploit

**Comment by CallMehMerel** [comment]
I dont 100% know if that would be the case or not since using steelhax doesnt help my case, but the exploit could be the problem still (not sure if using secondary exploits would change anything)

**Comment by Freshpep** [comment]
i'd just use rpwng to install oot3dhax or similar, then use that for themely or other hbl apps

**Comment by CallMehMerel** [comment]
I was going to try oot3dhax, but I ended up getting steelhax and none of the apps worked at all, so I didnt think using secondary exploits like oot3dhax made a difference

**Comment by memeToasty** [comment]
I dont have Ocorina of time so i cant use ootd hax and other secondarys arent updated yet

**Comment by CallMehMerel** [comment]
Oh I see. Guess I'll try and use oot3dhax and report back when Im done

**Comment by papermario69** [comment]
Yeah I'm getting a yellow screen with stickerhax. I used sploit installer from launcher. I'm on O3DS 11.5 USA

**Comment by papermario69** [comment]
If anyone does find a fix, please mention it

**Comment by memeToasty** [comment]
Is there a Solution in sight?

**Comment by ErmanSayin** [comment]
@memeToasty the problem is in the exploit & payloads. nothing I can do unless they're fixed by their authors

**Comment by AlwayzNick** [comment]
Lets hope so i tried oot3dshax but its stille the same

**Comment by papermario69** [comment]
So does anyone know if there is a payload in development?

**Comment by memeToasty** [comment]
http://gbatemp.net/threads/release-anemone3ds-a-complete-theme-and-splash-manager-for-your-3ds.482804/

This Manager works for 11.5 RPwnG

**Comment by ErmanSayin** [comment]
can anyone verify if v1.4.3 works with RPwnG?

**Comment by CallMehMerel** [comment]
v1.4.3 doesnt seem to work for me
